src/WebJobs.Script/Diagnostics/FileWriter.cs

```diff
@@ -127,56 +123,6 @@ public override void Flush()
             }
         }
 
-        public override void Trace(TraceEvent traceEvent)
-        {
-            if (traceEvent == null)
-            {
-                throw new ArgumentNullException(nameof(traceEvent));
-            }
-
-            object value;
-            if (traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyIsSystemTraceKey, out value)
-                && value is bool && (bool)value)
-            {
-                // we don't want to write system traces to the user trace files
```

Comment:
How is this filtering handled now - not logging user logs to host, and not logging system logs to user?

Reply:
When you write a log, you include these same properties in the state that's passed in. Then FileLogger behaves the same way as before -- just looking at the State since there's no concrete properties like there were in TraceWriter.